### ISSUE TITLE
Calibration and Uncertainty Characterization for Ultra-Wideband Two-Way-Ranging Measurements. (arXiv:2210.05888v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#23142]
Calibration and Uncertainty Characterization for Ultra-Wideband Two-Way-Ranging Measurements. (arXiv:2210.05888v1 [cs.RO])<br>
https://ift.tt/BOZrtAp<br>
<p>Ultra-Wideband (UWB) systems are becoming increasingly popular for indoor localization, where range measurements are obtained by measuring the time-of-flight of radio signals. However, the range measurements typically suffer from a systematic error or bias that must be corrected for high-accuracy localization. In this paper, a ranging protocol is proposed alongside a robust and scalable antenna-delay calibration procedure to accurately and efficiently calibrate antenna delays for many UWB tags. Additionally, the bias and uncertainty of the measurements are modelled as a function of the received-signal power. The full calibration procedure is presented using experimental training data of 3 aerial robots fitted with 2 UWB tags each, and then evaluated on 2 test experiments. A localization problem is then formulated on the experimental test data, and the calibrated measurements and their modelled uncertainty are fed into an extended Kalman filter (EKF). The proposed calibration is shown to yield an average of 46% improvement in localization accuracy. Lastly, the paper is accompanied by an open-source UWB-calibration Python library, which can be found at https://ift.tt/OctQJ0h.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
